src/containers/Aya/Articles.js

```diff
@@ -61,8 +62,9 @@ export default class Aya extends Component {
             {name: 'description', content: 'Aya'},
             {name: 'keywords', content: 'aya'}
           ]}
+          heroStyles={heroStyles}
           page={page}
-          articlesTitle="Articles"
+          articlesTitle=""
```

Comment:
Is this correct to be blank

Reply:
Yeah, I don't want a title as it is in the HeroPanel